tests/rkt_gc_test.go

```diff
@@ -89,36 +89,55 @@ func TestGCAfterUnmount(t *testing.T) {
 	ctx := testutils.NewRktRunCtx()
 	defer ctx.Cleanup()
 
-	imagePath := patchImportAndFetchHash("inspect-gc-test-run.aci", []string{"--exec=/inspect --print-msg=HELLO_API --exit-code=0"}, t, ctx)
-	defer os.Remove(imagePath)
-	cmd := fmt.Sprintf("%s --insecure-options=image prepare %s", ctx.Cmd(), imagePath)
-	uuid := runRktAndGetUUID(t, cmd)
+	for _, rmNetns := range []bool{false, true} {
 
-	cmd = fmt.Sprintf("%s run-prepared %s", ctx.Cmd(), uuid)
-	runRktAndCheckOutput(t, cmd, "", false)
+		imagePath := patchImportAndFetchHash("inspect-gc-test-run.aci", []string{"--exec=/inspect --print-msg=HELLO_API --exit-code=0"}, t, ctx)
+		defer os.Remove(imagePath)
+		cmd := fmt.Sprintf("%s --insecure-options=image prepare %s", ctx.Cmd(), imagePath)
+		uuid := runRktAndGetUUID(t, cmd)
 
-	stage1MntPath := filepath.Join(ctx.DataDir(), "pods", "run", uuid, "stage1", "rootfs")
-	stage2MntPath := filepath.Join(stage1MntPath, "opt", "stage2", "rkt-inspect", "rootfs")
+		cmd = fmt.Sprintf("%s run-prepared %s", ctx.Cmd(), uuid)
+		runRktAndCheckOutput(t, cmd, "", false)
 
-	if err := syscall.Unmount(stage2MntPath, 0); err != nil {
-		t.Fatalf("cannot umount stage2: %v", err)
-	}
+		podDir := filepath.Join(ctx.DataDir(), "pods", "run", uuid)
+		stage1MntPath := filepath.Join(podDir, "stage1", "rootfs")
+		stage2MntPath := filepath.Join(stage1MntPath, "opt", "stage2", "rkt-inspect", "rootfs")
+		netnsPath := filepath.Join(podDir, "netns")
+		podNetNSPathBytes, err := ioutil.ReadFile(netnsPath)
```

Comment:
Oh right.

Btw should this be documented in https://github.com/coreos/rkt/blob/master/Documentation/devel/on-disk-format.md?